blackbox_repository/conversion_scripts/scripts/nasbench201_import.py

```diff
@@ -0,0 +1,212 @@
+import bz2
+import pickle
+
+import matplotlib.pyplot as plt
```

Comment:
Can you put matplotlib inside the plot function? (it can fail some flake8 tests)